src/slider.styles.ts

```diff
@@ -0,0 +1,174 @@
+import { css } from 'lit';
+import focusOutline from './styles/focus-outline.js';
+
+export default [
+  css`
+    ${focusOutline('.handle:focus-visible')}
+  `,
+  css`
+    /* The designs call for a bit more spacing than the default when vertical. */
+    glide-core-private-label[orientation='vertical']::part(private-tooltips) {
+      margin-block-end: var(--glide-core-spacing-base-xxs);
+    }
+
+    .slider-container {
```

Comment:
I wanted to name this simply `component`, but it's not at the root level. So that doesn't quite make sense given our other patterns. Open to suggestions!